scripts/aws/deploy.py

```diff
@@ -126,6 +126,8 @@ def update_stage(client, rest_api_id, stage_name, stage_settings):
                         help="Location of edxapp for request routing")
     parser.add_argument("--catalog-host", required=True,
                         help="Location of catalog IDA for request routing")
+    parser.add_argument("--enterprise-host", required=True,
```

Comment:
By making this required, you're going to break any deployment pipelines that aren't aware of this new argument. Since you're still testing out enterprise endpoints, I strongly suggest making this field optional, so that we can update deployment scripts in an incremental way.